pkgs/top-level/config.nix

```diff
@@ -50,6 +50,39 @@ let
       default = false;
     };
 
+    nameSourcesPrettily =  mkOption {
```

Comment:
```suggestion
    usePrettierSourceNames =  mkOption {
```

Reply:
Hm, but why? `nameSourcesPrettily` is shorter and means the same thing.

Reply:
It's shorter because it is more obscure.
verb-adjective-name is clearer than name-name-adverb.

On the other hand, it depends on the previous tristate vs boolean-plus-state discussion.